lib/std/os.zig

```diff
@@ -254,6 +254,101 @@ pub fn close(fd: fd_t) void {
     }
 }
 
+pub const FChmodError = error{
+    AccessDenied,
+    InvalidFileDescriptor,
```

Comment:
I believe this error should should be removed and same thing with FChownError.

Reply:
The rest of `os.zig` does the same as this

Reply:
I think it's the opposite.  Most don't have this error.  I think there's 1 case that does.  May have been a mistake that got past review.

Reply:
In this case though, `BADF` can be raised if a directory has been opened without the `.iterate` option. That isn't an API violation, right?

Reply:
It looks like that is an API violation:
https://github.com/ziglang/zig/blob/0fb26b03690eb327164bbc7a21ba2d69d3b1a5b8/lib/std/fs.zig#L577

Reply:
Yeah, maybe so. It feels wrong that that should be an unreachable, but if that's what the rest of std does then I'll change this once I see whether these CI checks are passing. @andrewrk What are your thoughts on that?